biztalk/core/audit-management-operations.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Audit Management Operations  | Microsoft Docs"
+description: How to Audit management Operations.
+author: "Pravakar Garnayak"
+ms.author: "pravagar"
+manager: "dougeby"
+ms.date: "01/10/2020"
+ms.topic: conceptual
+ms.prod: biztalk-server
+
+# optional metadata
+
+#ROBOTS:
+
+ms.reviewer: 
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: "biztalk-2020"
+
+---
+
+
+# How to Audit Management Operations
+
+## Overview
+
+**Starting with BizTalk Server 2020**, you can audit management operation on application artifacts like send port, receive port, receive locations, orchestrations, resources etc. Auditing of Suspend/resume/terminate operation on service instances is also possible.
+
+## Configure auditing
+
+In fresh installation, auditing is disabled by default. To enable auditing:
+
+ 1. Open the **BizTalk Server Administration** console, right-click the **BizTalk Group**, and select **Settings**.
+
+ 2. Check **Audit management operations**.
+
+ 3. By default, **10000** audit log entries are stored in audit log table. If you want to change this, change **Maximum number of audit entries** setting to desired number.
+
+ 4. Select **OK** to save your changes.
+
+ 5. Refresh **BizTalk Server Administration**, if you want to do any management operation in same session.
+
+   ![bts2020audit_configure](../core/media/configure-audit.png)
+
+## Viewing audit logs
+
+1. Ensure that BizTalk operational data service is configured.
+2. Open a browser and type  http://localhost/BizTalkOperationalDataService/AuditLogs .
+
+3. To fetch audit log entries between a date range use http://localhost/BizTalkOperationalDataService/AuditLogs?fromDate=2019-12-25T01:00:00&toDate=2020-01-10 . Supported date formats for filtering are : *yyyy-MM-dd* or *yyyy-MM-ddThh:mm:ss* .
```

Comment:
To fetch audit log entries between a date range use web address with fromDate and toDate. For example,  http://localhost/BizTalkOperationalDataService/AuditLogs?fromDate=2019-12-25T01:00:00&toDate=2020-01-10
Supported date formats for filtering are : *yyyy-MM-dd* or *yyyy-MM-ddThh:mm:ss*